src/rpc/util.cpp

```diff
@@ -900,7 +900,16 @@ std::string RPCArg::ToStringObj(const bool oneline) const
 
 std::string RPCArg::ToString(const bool oneline) const
 {
-    if (oneline && !m_oneline_description.empty()) return m_oneline_description;
+    if (oneline && !m_oneline_description.empty()) {
```

Comment:
small nit:

```diff
diff --git a/src/rpc/util.cpp b/src/rpc/util.cpp
index d21760d235..9b93a626b7 100644
--- a/src/rpc/util.cpp
+++ b/src/rpc/util.cpp
@@ -3,6 +3,7 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <chainparamsbase.h>
+#include <clientversion.h>
 #include <consensus/amount.h>
```

to fix a compilation error; otherwise LGTM now